src/main/groovy/edu/oregonstate/mist/api/Info.groovy

```diff
@@ -10,3 +10,10 @@ class Info {
     String commit
     String documentation
 }
+
+/**
+ * Wrapper object for returning info in a response
+ */
+class InfoResultObject {
+    Info meta
```

Comment:
Previously, the information object was returned without being wrapped in a result object of any kind.
The response must have a top-level object of data, errors, or meta (http://jsonapi.org/format/#document-structure).

Given that this information refers to information about the API implementation itself, the meta object seems like the best choice. I made this InfoResultObject so that every ResultObject response doesn't have an unnecessary null meta object.

I considered putting this information in a JSON API object (http://jsonapi.org/format/#document-jsonapi-object), however this object must be a top-level member. Doing this would violate the first requirement of having data, errors, or meta as a necessary top-level member.

Reply:
Nice found. I agree that the `meta ` field sounds like is the most proper field for these information compared to others.